src/main/java/core/basesyntax/service/impl/FruitTransactionServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitTranzactionDao;
+import core.basesyntax.dao.impl.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitTransactionService;
+import java.util.List;
+
+public class FruitTransactionServiceImpl implements FruitTransactionService {
```

Comment:
I would change functionality of this service to process transactions (after parsing).

What I mean (global flow of all services):
1) (ReadService) read transactions (List<String> transactions)
2) (DataParser/TransactionParser)parse transactions (pass this list to parser to get -> List<FruitTransaction> transactions)
3) (this class/service or create new one - TransactionProcessor)
pass this transactions list to this service (you have list of transactions with operations, there will be logic to do some operations with Storage)
4) ReportService -> createReport (using Storage data)
5) write this report as String to file using WriterService